### ISSUE TITLE
Cannot use up and down Arrow to control the visual focus in all drop down list

### ISSUE THREAD

**Issue by mathisscott** [vmware/clarity#1937]
Priority:
P2

URL: http://clarity-ks.surge.sh/alerts

Reproduced Steps:
1. Go to Alerts page
2. Click “Cog angle” button to drop down the list.
3. Use up and down Arrow to control the visual focus in the drop down list

Actual Results:
Using up and down Arrow cannot control the visual focus in the drop down list.

Expected Results:
1. Using up and down Arrow can control the visual focus in the drop down list
2. The visual focus should be in the drop down list when using up and down Arrow

Screen shot:

![image.png](https://images.zenhubusercontent.com/582f4cffb9dc3c6220222f57/f998b0fb-fb24-4869-a82d-17073cc81e80)

Violation of Section 508 Standards: 
1194.21(c) A well-defined on-screen indication of the current focus shall be provided that moves among interactive interface elements as the input focus changes. The focus shall be programmatically exposed so that assistive technology can track focus and focus changes. 

User Impact: Blind.

Note:
This bug occurred with all dropdown list in the website.

**Comment by mathisscott** [comment]
This is a component issue.

**Comment by gnomeontherun** [comment]
Needs to consider multi-level dropdowns as well.

**Comment by gnomeontherun** [comment]
This was resolved with the work on accessibility and #3521, closing as fixed.
